prog/pwm/fancontrol

```diff
@@ -107,6 +108,8 @@ function LoadConfig
 		[ -z "${AFCMINPWM[$fcvcount]}" ] && AFCMINPWM[$fcvcount]=0
 		AFCMAXPWM[$fcvcount]=`echo $MAXPWM |sed -e 's/ /\n/g' |egrep "${AFCPWM[$fcvcount]}" |cut -d'=' -f2`
 		[ -z "${AFCMAXPWM[$fcvcount]}" ] && AFCMAXPWM[$fcvcount]=255
+		AFCCHARCUR[$fcvcount]=$(${FCCHARCUR// /\n} |grep -E "${AFCPWM[$fcvcount]}" |cut -d'=' -f2)
+		[ -z "${AFCCHARCUR[$fcvcount]}" ] && AFCCHARCUR[$fcvcount]=LIN    # Defauts to linear characteristic curve
```

Comment:
Unless I am missing something, "LIN" defines linear and everything else defines square. I'd rather have a boolean or 0/1 value.